### ISSUE TITLE
The create commit message does not correctly list all contributor following D.O upgrade

### ISSUE THREAD

**Issue by alexpott** [unicorn-fail/dreditor#53]
For example if you press the create commit message button on  https://drupal.org/node/2036239 the message is "Issue #2036239 by kevee: Color contrast on toolbar sub-menu links." but it should be "Issue #2036239 by kevee, mgifford: Color contrast on toolbar sub-menu links."

**Comment by star-szr** [comment]
Indeed, unfortunately the fix for this (#45) was after the 1.2.0 release. I would like to see another release sometime in the coming week, what do you think @markcarver?

In the meantime, to get the fix please install the [development release](http://dreditor.org/development) for now.

**Comment by star-szr** [comment]
1.2.1 is out now and contains this fix and a couple others.
